### DISCUSSION TITLE
https proxy

### DISCUSSION THREAD

**Discussion by henkiewie** [alexjustesen/speedtest-tracker#356] [Q&A] [answered]
Hi,

I tried the latest sqlite version of your docker app.
I am using my own https: proxy (nginx proxy manager), but when I add my certicate I get this;
<img width="1753" alt="Screenshot 2023-01-25 at 10 44 59" src="https://user-images.githubusercontent.com/74558874/214531495-7b6c3718-5484-4b3b-ab01-cec5ad48dccb.png">
When I go to the url with https its working but on an invalid certificate.
All other webistes work fine on https. This is the first one I see with this kind of behavior.

Is there maybe a solution for that?

Regards,
Henk

NB: I also tried to add a server ID, but its states its unknown, but when I add the URL or the ID from the test result it accepts and goes to that server for the next test, but it does not show up in the dropdown box in the general tab after a refresh.

**Comment by thechesu** [comment]
Got the same issue.    It works fine using my vps ip and port number.

**Reply by henkiewie** [reply]
Someone tested this on a SWAG proxy and it worked ok. So maybe it's a Nginx Proxy manager problem with Laravel.

**Reply by alexjustesen** [reply]
It likely is the allowed proxies, testing a build later today that should resolve that issue. Stay tuned

**Comment by henkiewie** [comment, marked as answer]
Just installed the latest release. Problem with Nginx proxy manager is solved.

**Reply by alexjustesen** [reply]
Nice

**Reply by snakekick** [reply]
can someone confirm that the problem with nginx as proxy is fixed? with me it still looks faulty with version .17 like with the thread starter

**Reply by henkiewie** [reply]
Using NPM `v2.10.4 © 2023` and `Speedtest Tracker v0.11.17` website is still running fine.

<img width="1897" alt="Screenshot 2023-08-24 at 13 22 09" src="https://github.com/alexjustesen/speedtest-tracker/assets/74558874/1e54ccb9-6137-45e4-a615-5244b0317f8b">
<img width="627" alt="Screenshot 2023-08-24 at 13 23 50" src="https://github.com/alexjustesen/speedtest-tracker/assets/74558874/12f0d46d-05ae-4605-aae6-2b2809d2053c">

**Comment by snakekick** [comment]
Thank you @henkiewie, you are right, it works.
